src/geode/mesh/core/surface_mesh.cpp

```diff
@@ -228,6 +228,29 @@ namespace geode
         {
         }
 
+        SurfaceVerticesAroundVertex vertices_around_vertex(
+            const SurfaceMesh< dimension >& mesh, index_t vertex_id ) const
+        {
+            SurfaceVerticesAroundVertex result;
+            for( const auto& poly_vertex :
+                mesh.polygons_around_vertex( vertex_id ) )
+            {
+                auto candidate = mesh.polygon_vertex(

```

Comment:
```suggestion
                const auto next_candidate = mesh.polygon_vertex(
```
And rename the next candidate variable